### DISCUSSION TITLE
Is there a way to automatically sync a JSON file with the ink file?

### DISCUSSION THREAD

**Discussion by Hussein-Attie** [inkle/ink#657] [Game integration help]
Can I have a JSON file that gets updated whenever the ink file is saved? For context, I'm using unity and to my knowledge there's no way that unity can identify the ink file in a buildable way, so is there any extension or scripts that would help me? Is it even possible?

Thanks.

**Comment by joethephish** [comment]
You can use the ink Unity integration package, available either [here on GitHub](https://github.com/inkle/ink-unity-integration/releases) or from the [Unity Asset Store](https://assetstore.unity.com/packages/tools/integration/ink-unity-integration-60055). The integration package automatically builds the ink for you, so you can just load up the json from game code.
